src/main/java/sirius/biz/storage/layer1/replication/ForceReplicationJob.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.storage.layer1.replication;
+
+import sirius.biz.jobs.batch.SimpleBatchProcessJobFactory;
+import sirius.biz.jobs.params.LocalDateParameter;
+import sirius.biz.jobs.params.Parameter;
+import sirius.biz.jobs.params.SelectStringParameter;
+import sirius.biz.process.ProcessContext;
+import sirius.biz.process.logs.ProcessLog;
+import sirius.biz.storage.layer1.ObjectStorage;
+import sirius.biz.storage.layer1.ObjectStorageSpace;
+import sirius.biz.storage.util.StorageUtils;
+import sirius.biz.tenants.TenantUserManager;
+import sirius.kernel.commons.Watch;
+import sirius.kernel.di.std.Part;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.nls.NLS;
+import sirius.web.security.Permission;
+
+import javax.annotation.Nonnull;
+import java.time.LocalDate;
+import java.util.Map;
+import java.util.function.Consumer;
+
+/**
+ * Forces the replication of a given {@link ObjectStorageSpace}.
+ * <p>
+ * This is most commonly used after a new replication target has been defined in order to copy the existing
+ * objects.
+ */
+@Register(framework = StorageUtils.FRAMEWORK_STORAGE)
+@Permission(TenantUserManager.PERMISSION_SYSTEM_ADMINISTRATOR)
+public class ForceReplicationJob extends SimpleBatchProcessJobFactory {
```

Comment:
Den Job gibts doch schon auf develop, wieso wird der hier als ganz neu dargestellt? Dadurch sieht man auch die eigentliche Änderung nicht.

In develop ist die Aufbewahrungszeit auch auf 14 Tage überschrieben, hier nicht.

Reply:
NVM - the persistence period was increased to 3 months - which happens to be the default